client/actions/types.js

```diff
@@ -17,3 +17,7 @@ export const DAYS_FETCH_DATA_SUCCESS = 'DAYS_FETCH_DATA_SUCCESS';
 export const DAILY_RESULT_HAS_ERRORED = 'DAILY_RESULT_HAS_ERRORED';
 export const DAILY_RESULT_IS_LOADING = 'DAILY_RESULT_IS_LOADING';
 export const DAILY_RESULT_FETCH_DATA_SUCCESS = 'DAILY_RESULT_FETCH_DATA_SUCCESS';
+
+export const SIMULATIONS_HAS_ERRORED = 'SIMULATIONS_HAS_ERRORED';
+export const SIMULATIONS_IS_LOADING = 'SIMULATIONS_IS_LOADING';
+export const SIMULATIONS_FETCH_DATA_SUCCESS = 'SIMULATIONS_FETCH_DATA_SUCCESS';
```

Comment:
I think a good pattern for that would be to create a single action and a generic reducer for both loading & error state. 

As for the fetch actions and future action type. I suggest that we stick with Redux convention by using plain string for the types. 

> It is a common convention that actions have a constant type that helps reducers (or Stores in Flux) identify them. We recommend that you use strings and not Symbols for action types, because strings are serializable, and by using Symbols you make recording and replaying harder than it needs to be.